packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -165,89 +165,108 @@ const StatisticsInfoModal = ({ is_mobile, is_statistics_info_modal_open, toggleS
-            is_drawer_open,
-            is_mobile,
-            is_statistics_info_modal_open,
-            onClearStatClick,
-            setActiveTabIndex,
-            toggleDrawer,
-            is_dialog_open,
-            onOkButtonClick,
-            onRunButtonClick,
-            onCancelButtonClick,
-            toggleStatisticsInfoModal,
-            onCloseDialog,
-        } = this.props;
-        const content = drawerContent({ active_index, is_drawer_open, setActiveTabIndex, ...this.props });
-        const footer = drawerFooter({ is_clear_stat_disabled, onClearStatClick });
+    React.useEffect(() => {
+        onMount();
+        return () => onUnmount();
+    }, [onMount, onUnmount]);
 
-        return (
-            <>
-                <div className={is_mobile && is_drawer_open ? 'run-panel__container--mobile' : undefined}>
-                    <Drawer
-                        className={!is_mobile ? 'run-panel__container' : undefined}
-                        contentClassName='run-panel__content'
-                        clear_stat_button_text={localize('Reset')}
-                        footer={!is_mobile && footer}
-                        is_clear_stat_disabled={is_clear_stat_disabled}
-                        is_mobile={is_mobile}
-                        is_open={is_drawer_open}
-                        onClearStatClick={onClearStatClick}
-                        toggleDrawer={toggleDrawer}
-                        zIndex={popover_zindex.RUN_PANEL}
-                    >
-                        {content}
-                    </Drawer>
-                    {is_mobile && <MobileDrawerFooter />}
-                </div>
-                <Dialog
-                    title={dialog_options.title}
-                    is_visible={is_dialog_open}
-                    cancel_button_text={dialog_options.cancel_button_text || localize('Cancel')}
-                    onCancel={onCancelButtonClick}
-                    confirm_button_text={dialog_options.ok_button_text || localize('OK')}
-                    onConfirm={onOkButtonClick || onCloseDialog}
-                    is_mobile_full_width={false}
-                    className={'dc-dialog__wrapper--fixed'}
-                    has_close_icon
-                >
-                    {dialog_options.message}
-                </Dialog>
-                <SelfExclusion
-                    onRunButtonClick={onRunButtonClick}
-                    onCancelButtonClick={this.props.resetSelfExclusion}
-                />
-                <StatisticsInfoModal
+    const content = drawerContent({
+        active_index,
+        currency,
+        is_drawer_open,
+        is_mobile,
+        lost_contracts,
+        number_of_runs,
+        setActiveTabIndex,
+        toggleStatisticsInfoModal,
+        total_payout,
+        total_profit,
+        total_stake,
+        won_contracts,
+    });
+    const footer = drawerFooter({ is_clear_stat_disabled, onClearStatClick });
```

Comment:
if you think it's better using jsx please update drawerFooter to DrawerFooter
```suggestion
    const footer = ()=>(<DrawerFooter  
                                         is_clear_stat_disabled=is_clear_stat_disabled
                                          onClearStatClick =onClearStatClick 
                                          /> )
```

Reply:
Updated `drawerFooter` to `DrawerFooter` and `drawerContent` to `DrawerContent` 👍